### DISCUSSION TITLE
not getting ibax despite reference

### DISCUSSION THREAD

**Discussion by Mekmevin12** [IBAX-io/go-ibax#71] [Rewards]
Why did I not get any ibax even though I was a reference to my friend?

**Comment by HalloKyle** [comment]
> Why did I not get any ibax even though I was a reference to my friend?

You won't get IBAX points when you ref friends but you will get IBXC coins directly!
IBAX points is used for redeem IBXC coins.

**Comment by Emirxt** [comment]
Arkadaşınız kaydolduktan sonra 1-2 tane görev yapsın IBXC GELİCEKTİR
